### PR TITLE
add swift scripts as per existing kotlin scripts. 4.opreturn.swift

### PR DESCRIPTION
**Pull request by geigerzaehler242**
added corresponding swift scripts as per the kotlin scripts. 4.opreturn.swift

### CODE REVIEW

**Comment by thunderbiscuit** [comment]
Thanks for the contribution!

I'll try and review it this week.

**Comment by reez** [comment]
@geigerzaehler242 hi! excited to have your contributions! 

I've made a few myself to this repo related to Swift so I'd love to help review and get your contributions merged in. To make the process as clear and easy as possible when reviewing and merging is there any chance you can break up this PR into multiple PR's focused on one specific item each?

For example, this PR has 7 separate items so if you could create a separate PR for each of the 7 items then we can start reviewing them in sequential order possibly.

Happy to help you in any way needed, and looking forward to reviewing, thanks!

**Comment by reez** [comment]
> @geigerzaehler242 hi! excited to have your contributions!
> 
> I've made a few myself to this repo related to Swift so I'd love to help review and get your contributions merged in. To make the process as clear and easy as possible when reviewing and merging is there any chance you can break up this PR into multiple PR's focused on one specific item each?
> 
> For example, this PR has 7 separate items so if you could create a separate PR for each of the 7 items then we can start reviewing them in sequential order possibly.
> 
> Happy to help you in any way needed, and looking forward to reviewing, thanks!

@geigerzaehler242 I've reviewed `4.opreturn.main.swift` so far and its looking good, I have a few comments that I'll hold off on adding until I know if you are going to keep this PR or split the 7 items into separate PR's. Nice work so far though!

**Comment by geigerzaehler242** [comment]
> > @geigerzaehler242 hi! excited to have your contributions!
> > I've made a few myself to this repo related to Swift so I'd love to help review and get your contributions merged in. To make the process as clear and easy as possible when reviewing and merging is there any chance you can break up this PR into multiple PR's focused on one specific item each?
> > For example, this PR has 7 separate items so if you could create a separate PR for each of the 7 items then we can start reviewing them in sequential order possibly.
> > Happy to help you in any way needed, and looking forward to reviewing, thanks!
> 
> @geigerzaehler242 I've reviewed `4.opreturn.main.swift` so far and its looking good, I have a few comments that I'll hold off on adding until I know if you are going to keep this PR or split the 7 items into separate PR's. Nice work so far though!

is it ok just to review all 7 additions as one PR this time? in the future if i add more scripts i will break them up into individual pr's.

**Comment by thunderbiscuit** [comment]
I think it's worth doing it right. Let us know if you need help breaking them up or if you're good to go (we don't assume a lot of git knowledge, so it's totally cool if you need pointers).

You don't need to close this PR, you can just force-push a new version that has only 1 script added. Note that you're also missing the changes necessary in the readme.

File: swift/4.opreturn.main.swift [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    let progress = LogProgress()
+    try wallet.sync(blockchain: blockchain, progress: progress)
+
+    let balance = try wallet.getBalance().total
+
+    if balance == 0 {
+        print("Wallet balance is 0, please add funds!")
+    }
+
+    let message = "Credit Suisse purchased by UBS."
```
Comment (reez) [inline swift/4.opreturn.main.swift:41]: What do you think about changing the message text to ["Working with friends is a happy way to live."](https://github.com/thunderbiscuit/bitcoindevkit-scripts/blob/c85c1cdf3062cc57e57013c2585aa5caae25e3c7/kotlin/4.opreturn.main.kts#L34) just to match the Kotlin script?

**Review by reez — COMMENTED** [review]
Looking great! 2 small comments/requests in the review.

And then also can you change the filename to omit "main"? i.e. `4.opreturn.swift` instead of `4.opreturn.main.swift`

**Review by reez — APPROVED** [review]
LGTM! Nice work-

**Comment by geigerzaehler242** [comment]
as soon as you merge it i'll create the next branch and submit a new pr.

**Comment by thunderbiscuit** [comment]
I squashed those commits into 1. Thanks for the PR!

**Pull request merged**